source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -42,6 +42,7 @@
     <ProjectReference Include="..\Calamari.Aws\Calamari.Aws.csproj" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
+    <PackageReference Include="NETStandard.Library" Version="2.0.3" />
```

Comment:
Required for net framework to use System.Runtime.InteropServices.RuntimeInformation.ProcessArchitecture in RequiresNonARM64BitAttribute.cs